.github/workflows/cherry-pick-milestoned-prs.yml

```
@@ -0,0 +1,262 @@
+          branch_count=0
+          max_branches=2  # Limit to 2 most recent release branches
+
+          current_major=$MAJOR
+          current_minor=$MINOR
+
+          while [ $branch_count -lt $max_branches ]; do
+            branch="release/$current_major.$current_minor"
+            response=$(curl -s -o /dev/null -w "%{http_code}" \
+              -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+              "https://api.github.com/repos/${{ github.repository }}/branches/$branch")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)